### ISSUE TITLE
rlinks: support wildcard registrations and subscription

### ISSUE THREAD

**Issue by om26er** [crossbario/crossbar#1959]
Rlinks currently don't support wildcard subscriptions and registrations. I think it's an important missing feature and something we should implement.

**Comment by om26er** [comment]
@oberstet can you share your thoughts on the implementation design for this one ?

**Comment by oberstet** [comment]
I haven't thought about this at the code/design level and, fwiw, I would first solve https://github.com/crossbario/crossbar/issues/1903, as otherwise testing will become very hard

**Comment by om26er** [comment]
> I would first solve https://github.com/crossbario/crossbar/issues/1903, as otherwise testing will become very hard
Agreed. I will start working on that and will continue the conversation on that issue. Once we have some sort of automation in place, then will come back to this issue

**Comment by oberstet** [comment]
also, there are some existing automated tests which are currently deactivated (because sth with the CI wasn't working):

https://github.com/crossbario/crossbar/blob/5c3d0e44e7e2b72357b7364ab8e945e08a994b27/.github/workflows/main.yml#L156
https://github.com/crossbario/crossbar/tree/master/test
